readhomer_atlas/library/migrations/0011_add_place_named_entities.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.10 on 2020-04-30 22:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("library", "0010_metricalannotation"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="namedentity",
+            name="kind",
+            field=models.CharField(
+                choices=[("person", "Person"), ("person", "Place")], max_length=6
```

Comment:
second "person" should be "place"

Reply:
(actually I guess the error is above in the constants.py file)

Reply:
I didn't know you were looking at the PR; I did push a later commit and migration to fix; will likely get the PR out of WIP to reviewable quite soon 😉